de.metas.issue.tracking.github/pom.xml

```diff
@@ -0,0 +1,65 @@
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <parent>
+        <artifactId>de.metas.parent</artifactId>
+        <groupId>de.metas</groupId>
+        <version>10.0.0</version>
+        <relativePath>../pom.xml</relativePath>
+    </parent>
+    <modelVersion>4.0.0</modelVersion>
+
+    <groupId>de.metas.issue.tracking</groupId>
+    <artifactId>de.metas.issue.tracking.github</artifactId>
+
+    <properties>
+        <migration-sql-basedir></migration-sql-basedir>
+        <metasfresh.version>[1,10.0.0]</metasfresh.version>
+    </properties>
+
+    <dependencies>
+        <dependency>
+            <groupId>org.slf4j</groupId>
+            <artifactId>slf4j-api</artifactId>
+        </dependency>
+
+        <dependency>
+            <groupId>org.springframework</groupId>
+            <artifactId>spring-context</artifactId>
+        </dependency>
+
+        <dependency>
+            <groupId>com.fasterxml.jackson.core</groupId>
+            <artifactId>jackson-annotations</artifactId>
+        </dependency>
+
+        <dependency>
+            <groupId>org.projectlombok</groupId>
+            <artifactId>lombok</artifactId>
+            <scope>provided</scope>
+        </dependency>
+
+        <dependency>
+            <groupId>org.springframework</groupId>
+            <artifactId>spring-web</artifactId>
+        </dependency>
+
+        <dependency>
```

Comment:
basically, it will not be included in the class path of a module that's dependent on this one. Or in other words, if let's say there is a module ```Test``` that has a dependency on ```de.metas.issue.tracking.github```, the optional dependencies from ```..github``` will not be added in class path for ```Test```.
Anyway, I'll remove the dependency as I've dropped the ```.properties ``` file approach somewhere down the road and it's not needed anymore.